src/OMSimulatorLib/Model.h

```diff
@@ -75,7 +75,7 @@ namespace oms
     oms_status_enu_t exportSnapshot(const ComRef& cref, char** contents);
     oms_status_enu_t exportSSVTemplate(const ComRef& cref, const std::string& filename);
     oms_status_enu_t exportSSMTemplate(const ComRef& cref, const std::string& filename);
-    oms_status_enu_t importFromSSD(const pugi::xml_node& node);
+    oms_status_enu_t importFromSnapshot(const pugi::xml_node& oms_snapshot);
```

Comment:
Maybe we shouldn't use the snapshot node, but a `unordered_map<filename, xml_node>`. @arun3688 what do you think?

Reply:
We can use `unordered_map` , but I like the current `<oms_snapshot>`, Is it because of the performance issue ? or some other advantages?

Reply:
this is done in this commit https://github.com/OpenModelica/OMSimulator/pull/961/commits/b307b78706008dba6e31fbeb34ea8fb7407fd0a3